modules/accounts.js

```diff
@@ -104,12 +105,31 @@ Accounts.prototype.generateAddressByPublicKey = function(publicKey) {
  * @param {function} cb - Callback function
  */
 Accounts.prototype.getAccount = function(filter, fields, cb, tx) {
+	const publicKey = filter.publicKey;
 	if (filter.publicKey) {
 		filter.address = self.generateAddressByPublicKey(filter.publicKey);
 		delete filter.publicKey;
 	}
 
-	library.logic.account.get(filter, fields, cb, tx);
+	if (typeof fields === 'function') {
```

Comment:
Same here, it's not needed to be backported, we collected all the info and this piece of code only adds complexity to the codebase.